<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-42J99NO" failures="3" tests="22" name="Surefire test" time="9.16" errors="0" timestamp="2023-10-18T23:53:34 EEST">
  <testcase classname="weare.api.testing.comment.CreateCommentTest" name="createComment_Successful" time="0.121"/>
  <testcase classname="weare.api.testing.comment.DeleteCommentTest" name="deleteComment_Successful" time="1.052"/>
  <testcase classname="weare.api.testing.comment.EditCommentTest" name="editComment_Successful" time="0.065"/>
  <testcase classname="weare.api.testing.comment.FindAllCommentsTest" name="findAllComments_Successful" time="0.032">
    <failure type="java.lang.AssertionError" message="The array size is more than or equal to 1 expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The array size is more than or equal to 1 expected [true] but found [false]
at weare.api.testing.comment.FindAllCommentsTest.findAllComments_Successful(FindAllCommentsTest.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- findAllComments_Successful -->
  <testcase classname="weare.api.testing.comment.LikeDislikeCommentTest" name="likeDislikeComment_Successful" time="0.062"/>
  <testcase classname="weare.api.testing.connection.ApproveRequestTest" name="approveRequest_successful" time="0.138"/>
  <testcase classname="weare.api.testing.connection.GetUserRequestsTests" name="getUserRequestTest" time="0.13"/>
  <testcase classname="weare.api.testing.connection.SendRequestTest" name="sendRequest_successful" time="0.143"/>
  <testcase classname="weare.api.testing.post.CreatePostTest" name="createPost_Successful" time="0.156"/>
  <testcase classname="weare.api.testing.post.DeleteAPostTest" name="deletePost_Successful" time="0.028"/>
  <testcase classname="weare.api.testing.post.EditPostTest" name="editPost_Successful" time="0.029"/>
  <testcase classname="weare.api.testing.post.LikeDislikeAPostTest" name="likeAndDislikePost_Successful" time="0.058"/>
  <testcase classname="weare.api.testing.skill.CreateSkillTests" name="createSkill_Successful" time="0.053"/>
  <testcase classname="weare.api.testing.skill.DeleteSkillTests" name="deleteSkill_Successful" time="0.022"/>
  <testcase classname="weare.api.testing.skill.GetAllSkillsTests" name="newlyCreatedSkill_returned_whenGetAllSkills" time="0.023"/>
  <testcase classname="weare.api.testing.skill.GetOneSkillTests" name="getOneSkillById_Successful" time="0.016"/>
  <testcase classname="weare.api.testing.skill.UpdateSkillTests" name="updateSkill_Successful" time="0.02"/>
  <testcase classname="weare.api.testing.users.GetProfilePostsTests" name="getUserPosts_Successful" time="0.042"/>
  <testcase classname="weare.api.testing.users.GetUserByIdTests" name="getUserById_Successful" time="0.014">
    <failure type="java.lang.AssertionError" message="Incorrect status code. expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: Incorrect status code. expected [200] but found [404]
at base.BaseTestSetup.isResponse200(BaseTestSetup.java:52)
at weare.api.testing.users.GetUserByIdTests.getUserById_Successful(GetUserByIdTests.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- getUserById_Successful -->
  <testcase classname="weare.api.testing.users.GetUsersTests" name="getUsers_Successful" time="0.043"/>
  <testcase classname="weare.api.testing.users.RegisterUserTests" name="registerUserSuccessfully" time="0.14"/>
  <testcase classname="weare.api.testing.users.UpdateUserExpertiseTests" name="updateUserExpertise_Successful" time="0.165">
    <failure type="java.lang.AssertionError" message="Incorrect status code. expected [200] but found [500]">
      <![CDATA[java.lang.AssertionError: Incorrect status code. expected [200] but found [500]
at base.BaseTestSetup.isResponse200(BaseTestSetup.java:52)
at weare.api.testing.users.UpdateUserExpertiseTests.updateUserExpertise_Successful(UpdateUserExpertiseTests.java:31)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- updateUserExpertise_Successful -->
</testsuite> <!-- Surefire test -->
